libs/sdk-ui-ext/src/internal/utils/embeddingInsightViewCodeGenerator/insightViewCodeGenUtils.ts

```diff
@@ -30,7 +30,7 @@ export function insightViewAdditionalTransformations(
                 package: idRef,
             },
             transformation: (obj) => {
```

Comment:
Are those `insightViewAdditionalTransformations` needed anymore since we do the string shortcut?

Reply:
deleted